host/xtest/stats.c

```diff
@@ -209,6 +218,98 @@ static int stat_memleak(int argc, char *argv[])
 	return close_sess(&ctx, &sess);
 }
 
+static int stat_loaded_ta(int argc, char *argv[])
+{
+	TEEC_Context ctx = { };
+	TEEC_Session sess = { };
+	TEEC_Result res = TEEC_ERROR_GENERIC;
+	uint32_t eo = 0;
+	TEEC_Operation op = { };
+	struct ta_dump_stats *stats = NULL;
+	size_t stats_size_bytes = 0;
+	size_t n = 0;
+
+	UNUSED(argv);
+	if (argc != 1)
+		return usage();
+
+	open_sess(&ctx, &sess);
+
+	op.paramTypes = TEEC_PARAM_TYPES(TEEC_MEMREF_TEMP_OUTPUT,
+					 TEEC_NONE, TEEC_NONE, TEEC_NONE);
+	res = TEEC_InvokeCommand(&sess, STATS_CMD_TA_STATS, &op, &eo);
+	if (res != TEEC_ERROR_SHORT_BUFFER) {
+		if (res == TEEC_ERROR_NOT_SUPPORTED)
+			printf("Build OP-TEE OS without CFG_TA_STATS ?= y\n");
```

Comment:
I'm not sure this is needed.

Reply:
This is to reminder user to have "CFG_TA_STATS ?= y" build if they want to run this cmd. I think it's nice to have it.

Reply:
It litters the code in my opinion.

Reply:
okay, done with removing that line printf().